libshaderc_util/include/libshaderc_util/compiler.h

```diff
@@ -97,6 +159,10 @@ class Compiler {
   // from the shader text. Any #include directives are parsed with the given
   // includer.
   //
+  // The initializer parameter must be a valid GlslInitializer object.
+  // Acquire will be called on the initializer, and cleaned up
```

Comment:
"cleaned up" is ambiguous: better to say that `Release` will be called when compilation is finished.